apps/arena/priv/config.json

```diff
@@ -500,6 +500,18 @@
           "execute_multiple_times": true
         }
       }
+    },
+    {
+      "name": "polymorph_effect",
+      "duration_ms": 9000,
+      "remove_on_action": true,
+      "one_time_application": true,
+      "effect_mechanics": {
+        "polymorph": {
```

Comment:
Why do you need this? Wouldn't an empty list be the same?

Reply:
We don't need it you're right, i was replicating what the invisible effect have but this isn't necessary, deleted